### DISCUSSION TITLE
Add Folders

### DISCUSSION THREAD

**Discussion by CasanovaSan** [amnweb/yasb#67] [Ideas]
I would really like the ability to have folders like the ones in droptop

![image](https://github.com/user-attachments/assets/0241fd5f-eae8-40af-a806-92afb705f101)

**Comment by amnweb** [comment]
Please tell me more, this is the user's home folder, right?
What's happened when you hover on any folder, it shows subfolders?
What's happened when you click on a file?
Also, is there any config to include and exclude folders that you don't need to show? Does it show OS hidden folders?

**Reply by CasanovaSan** [reply]
Yes thats the user folder

Yes if you click a  subfodlers it opens another popup with the subfolder

If you click on a file that isn't a folder it opens it

You can change what folder you want, for example i can change the User folder to desktop or any folder i have on my pc

Yes there is an option to show os hidden files or not show them

You can't hide particular folders tho with droptop, they have to be hidden by os

You can pick whatever colors you want for the folder button and emojis too


If this would ever be added, pls add a grid view with resizeable icons (pls make it per folder) since droptop only got a list view

**Comment by amnweb** [comment]
Ok got it, not sure about this 
`If this would ever be added, pls add a grid view with resizeable icons (pls make it per folder) since droptop only got a list view`

**Reply by CasanovaSan** [reply]
it doesn't have to mendatory, but as an option

And i know this would be even more complicated, but what about the idea of dragging the files out of that panel to the desktop and actually doing it? like actually being able to drag the file out of the yasb bar folder panel to the desktop or apps, etc, or even right clicking them and getting options to delete them or stuff like that, making it actually functional and not just a way to view files and open them

**Reply by amnweb** [reply]
This will require rewriting the entire code, and I think this is not what YASB should be. About drop down menu I will add something like this


https://github.com/user-attachments/assets/bac40b7b-e6d5-4376-af08-f1ee5a4ca622
